spec/acceptance/class_spec.rb

```diff
@@ -24,7 +24,7 @@ class { 'splunk_hec': url => "http://httpbin.org/post", token => '', record_even
     end
 
     it 'Successfully sends data to an http endpoint' do
-      run_shell('cat /etc/puppetlabs/code/environments/production/modules/splunk_hec/examples/orchestrator_metrics.json | puppet splunk_hec --sourcetype puppet:summary --saved_report')
+      run_shell('cat /etc/puppetlabs/code/environments/production/modules/splunk_hec/examples/orchestrator_metrics.json | puppet splunk_hec --sourcetype puppet:summary')
```

Comment:
Why this change?

Reply:
The `--saved_report` option will submit the request immediately to the splunk instance configured in splunk_hec.yaml. It will not work currently as that file is not yet fully configured with the credentials for the splunk container we are running. I'm planning a PR right after this one to add this back and more acceptance tests that actually communicate with the Splunk container. I could also add it for just this test in this PR as well if we feel that changing this test feels janky. Happy to go either way.

Reply:
Ah ok, that's fine. Was just curious.